tools/crm_simulate.c

```diff
@@ -26,11 +26,44 @@
 #include <crm/pengine/status.h>
 #include <pacemaker-internal.h>
 
+/* show_scores and show_utilization can't be added to this struct.  They
+ * actually come from include/pcmki/pcmki_scheduler.h where they are
+ * defined as extern.
+ */
```

Comment:
FYI one of my "mindless" to-dos is converting show_scores/utilization to be pe_flag_* bits instead, but I'm not 100% sure that's the best replacement. When we convert crm_simulate to use output objects it should become obvious (they cause libpacemaker to print certain things to stdout)